libs/net/dal/Migrations/20230707180726_1.0.43.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using TNO.DAL;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace TNO.DAL.Migrations
+{
+    /// <inheritdoc />
+    public partial class _1043 : SeedMigration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            PreUp(migrationBuilder);
+            migrationBuilder.AlterColumn<long>(
```

Comment:
I don't understand why there are so many changes to the Minister table?

Reply:
I'm not sure what happened here either, debugging it. I redid the migration after pulling in changes and it spat this out.

Reply:
@CelloLiu @kylgarmor 

You need to correctly follow this process when removing invalid migrations that occur when another PR creates one.
Before getting the latest code and merging.
`make db-rollback n={prior version}`
`make db-remove`
Manually undo all changes to the `TNOContextModelSnapshot.cs` with git.
Get the latest code changes.
`make db-update`
`make db-add n={your new version}`
`make db-update`
Now test to make sure your rollback works
`make db-rollback n={prior version}`
`make db-update`